src/applications/financial-status-report/components/PreSubmitSignature.jsx

```diff
@@ -101,6 +101,6 @@ const PreSubmitSignature = ({ formData, showError }) => {
   );
 };
 export default {
-  required: true,
+  required: false,
```

Comment:
Why would this not be required? In my XP they usually are but I am not sure of the scope of this work either

Reply:
well it is but were not at the stage to implement validation on the checkbox until submission data is working so disabling for now

Reply:
Roger that makes sense